setup.cfg

```diff
@@ -10,7 +10,14 @@ description_file = README.md
 description_content_type = text/markdown
 license_files = LICENSE.md
 version = attr: ansible_sdk.__version__
-
+classifiers = [
```

Comment:
I don't believe the format of this is correct for a `setup.cfg` (but is within `setup.py`).